app/api/routes.py

```diff
@@ -179,6 +184,32 @@ def get_resources():
     return standardize_response(payload=dict(data=resource_list, **pagination_details))
 
 
+def search_results():
+    term = request.args.get('q', '', str)
+    page = request.args.get('page', 0, int)
+    query = Resource.query
+
+    search_result = index.search(f'{term}', {
+        'page': page,
+    })
```

Comment:
Clients are allowed to pass in a query param called `page_size`, and we should honor that here.

Should add in something like
```python
'hitsPerPage': request.args.get('page_size')
```
but default to the `per_page` specified in the Configs file if they don't pass in this param

Reply:
To make this more readable, let's make the `request.args.get` call above and store it in a variable like we do with `page`